docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -2019,12 +2307,104 @@ Keys:
 
 **Recommended usage**
 
-``` xml
+```xml
 <send_crash_reports>
     <enabled>true</enabled>
 </send_crash_reports>
 ```
 
+## ssh_server
+
+Configuration for writing the public part of a host key to a known_hosts file on the SSH client side on the first connect.
```

Comment:
I applaud your effort to make the comment in config.xml easier to read. Unfortunately, while it has been hard to understand, it is now impossible to understand. Let's just retain the original comment from config.xml.